src/qt/intro.cpp

```diff
@@ -130,18 +131,18 @@ Intro::Intro(QWidget *parent, uint64_t blockchain_size, uint64_t chain_state_siz
     );
     ui->lblExplanation2->setText(ui->lblExplanation2->text().arg(PACKAGE_NAME));
 
-    uint64_t pruneTarget = std::max<int64_t>(0, gArgs.GetArg("-prune", 0));
-    if (pruneTarget > 1) { // -prune=1 means enabled, above that it's a size in MB
+    int64_t prune_target_mib = std::max<int64_t>(0, gArgs.GetArg("-prune", 0));
+    if (prune_target_mib > 1) { // -prune=1 means enabled, above that it's a size in MiB
         ui->prune->setChecked(true);
         ui->prune->setEnabled(false);
     }
-    ui->prune->setText(tr("Discard blocks after verification, except most recent %1 GB (prune)").arg(pruneTarget ? pruneTarget / 1000 : DEFAULT_PRUNE_TARGET_GB));
```

Comment:
In commit "util: Add PruneMiBtoGB() function" (162dafaaa9d47e7d40db09040a265bccf975be20)

Maybe update commit description to note slight change in label text: now displays GB number instead of GiB number.

I think there are no other changes in behavior in this commit, or at least I don't see any other changes.

Reply:
Done.



> I think there are no other changes in behavior in this commit, or at least I don't see any other changes.

Correct.